firebase/README.md

```diff
@@ -0,0 +1,62 @@
+## How to send web push notifications to a mobile device with Firebase
+
+This example project shows you how to send TalkJS's web push notifications to a mobile device with Firebase.
+
+> [!TIP] > [Download this example project as a zip file](https://github.com/talkjs/talkjs-examples/releases/latest/download/firebase.notifications-example.zip)
+
+## Prerequisites
+
+To run this example project, you need:
+
+- The [Firebase CLI](https://firebase.google.com/docs/cli#install_the_firebase_cli)
+- A [TalkJS account](https://talkjs.com/dashboard/login)
+
+## How to run the example
+
+1. Clone or [download this project](https://github.com/talkjs/talkjs-examples/releases/latest/download/firebase.notifications-example.zip).
+1. In the root directory of the project, run:
+   ```sh
+   firebase login
+   ```
+   if you are not already logged in, and log in to your account
+1. Set up Firebase hosting:
+   ```sh
+   firebase init hosting
+   ```
+   Select to either use an existing Firebase project or create a new one. If you create a new one, or you haven't yet set up Firebase Cloud Messaging in the TalkJS dashboard, see [Configure Firebase Cloud Messaging](https://talkjs.com/docs/Features/Notifications/Mobile_Push_Notifications/Configure_FCM/) in our docs.
+1. Answer the prompted questions:
+   1. Select to use the default (`public`) as your public directory
+   1. Select Yes for "Configure as a single-page app?"
+   1. Choose whether to set up automatic builds and deploys with GitHub
+   1. Select No for "File public/index.html already exists. Overwrite?"
+1. Once initialization is complete, open the [Firebase console](http://console.firebase.google.com/) and select your project.
+1. On the **Project Overview** page, click the **+** icon to add a new app. For the platform, select **Web**.
+1. Pick a name for your app.
+1. After registering the app name, you will see some code samples for adding the Firebase SDK. Copy the `firebaseConfig` JavaScript object. It will look something like this:
+   ```json
+   {
+     "apiKey": "<API_KEY>",
+     "authDomain": "<APP_NAME>.firebaseapp.com",
+     "projectId": "<APP_NAME>",
+     "storageBucket": "<APP_NAME>.firebasestorage.app",
+     "messagingSenderId": "<SENDER_ID>",
+     "appId": "<APP_ID>"
```

Comment:
```suggestion
     "appId": "<FIREBASE_APP_ID>"
```
On line 50 we use `APP_ID` in reference to the TalkJS app ID. So we should rename either instance so that someone isn't confused.